crml/staking/src/rewards/mod.rs

```diff
@@ -136,16 +145,21 @@ where
 		validator_commission_stake_map: &[(Self::AccountId, Perbill, Exposure<Self::AccountId, Self::Balance>)],
 		era: EraIndex,
 	) {
-		// Calculate the accumulated tx fee reward split
-		let fee_payout = TransactionFeePot::<T>::take();
 		// track historic era fee amounts
-		Self::note_fee_payout(fee_payout);
+		Self::note_fee_payout(TransactionFeePot::<T>::get());
 
-		if fee_payout.is_zero() || validator_commission_stake_map.len().is_zero() {
-			return;
+		if era.saturating_sub(Self::fiscal_era_start()) >= T::FiscalDuration::get() {
```

Comment:
might not need the extra storage field: `era % T::FiscalEraLength::get()`

Reply:
It would need that when I add "force_new_fiscal_era" for SUDO

Reply:
However, using % I can avoid writing to that extra storage when not needed